### PR TITLE
Add MCS Expert Filter front

### PR DESCRIPTION
**Pull request by lilian-houdelet**

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -268,6 +268,15 @@
   "download.message.studies": "studies",
   "download.message.others": "others",
   "connected": "Connected",
+  "maximumSectionCount": "Maximum number of sections",
+  "sectionCount": "Current number of sections",
+  "shuntCompensatorType": "Type",
+  "Capacitor": "Capacitor",
+  "Reactor": "Reactor",
+  "maxQAtNominalV": "Qmax available at nominal voltage",
```
Comment (dbraquart) [inline src/translations/en.json]:
```suggestion
  "maxQAtNominalV": "Available Qmax at nominal voltage",
```

File: src/translations/en.json [hunk 2]
```diff
@@ -268,6 +268,15 @@
...
+  "SwitchedOnMaxQAtNominalV": "Switch-on Q at nominal voltage",
+  "maxSusceptance": "Maximal susceptance available",
```
Comment (dbraquart) [inline src/translations/en.json]:
```suggestion
  "maxSusceptance": "Maximum available susceptance",
```

**Review by dbraquart — CHANGES_REQUESTED** [review]
code: ok
tests: Front works fine, but some translations are not conform

**Review by dbraquart — APPROVED** [review]
code: ok
tests: not done with last commit

File: src/components/dialogs/filter/expert/expert-filter-constants.ts [hunk 3]
```diff
@@ -239,6 +248,50 @@ export const FIELDS_OPTIONS = {
...
+        dataType: DataType.NUMBER,
+        inputType: 'number',
+    },
+    MAX_SUSCEPTANCE: {
+        name: FieldType.MAX_SUSCEPTANCE,
+        label: 'maxSusceptance',
+        dataType: DataType.NUMBER,
+        inputType: 'number',
+    },
+    SWITCHED_ON_MAX_SUSCEPTANCE: {
```
Comment (flomillot) [inline src/components/dialogs/filter/expert/expert-filter-constants.ts]: Same as in backend, why with MAX ?

**Review by flomillot — APPROVED** [review]

**Pull request merged**